src/main/java/frc/robot/subsystems/intake/commands/IntakeCargo.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems.intake.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.intake.Intake;
+
+import java.util.function.BooleanSupplier;
+
+/**
+ * Intakes only Cargo colored as the robot's alliance color.
+ */
+public class IntakeCargo extends CommandBase {
```

Comment:
`Intake` is enough, there is only one type of game piece.

Reply:
I'd rather not call a command by the name of it's subsystem.